core/src/test/scala/unit/kafka/zk/ZooKeeperTestHarness.scala

```diff
@@ -43,7 +43,7 @@ abstract class ZooKeeperTestHarness extends JUnitSuite with Logging {
   val zkSessionTimeout = 6000
   val zkMaxInFlightRequests = Int.MaxValue
 
-  protected val zkAclsEnabled: Option[Boolean] = None
+  protected def zkAclsEnabled: Option[Boolean] = None
```

Comment:
why this change? prefer val. If there is a problem with initialization order, consider lazy val.

Reply:
I need to initialize this value in one of the classes. made this def so that we can override in subclasses.